ls1intum/Apollon
# `Development`: Adapt tests to React 18

### Motivation and Context
This PR updates testing-library/react and modify test. Tests will run closer to what real users would experience when using Apollon.

### Steps for Testing
1. Clone the repo
2. Execute commands: `yarn install`, `yarn build` followed by `yarn start`
3. Observe application works as expected.
